hc/settings.py

```diff
@@ -12,6 +12,8 @@
 
 import os
 import warnings
+import sendgrid
+import dj_database_url
```

Comment:
Do we need to import this database here? I though it was for our production db. Shouldn't it be in our production.py?